packages/jitar/src/runtime/ProcedureRunner.ts

```diff
@@ -17,6 +17,10 @@ export default class ProcedureRunner implements Middleware
     // eslint-disable-next-line @typescript-eslint/no-unused-vars
     handle(fqn: string, version: Version, args: Map<string, unknown>, headers: Map<string, string>, next: NextHandler): Promise<unknown>
     {
-        return this.#runner.run(fqn, version, args, headers);
+        const result = this.#runner.run(fqn, version, args, headers);
+
+        headers.clear();
```

Comment:
Clearing the headers here might cause issues in case the results comes from a remote. If the remote sets response headers, they will get lost here. We need to discuss what we want to do with this.

We might also want to move this class to the middleware module (just like the CORS middleware)